### DISCUSSION TITLE
UR10e Robot in robot library

### DISCUSSION THREAD

**Discussion by HmarKabob** [visose/Robots#191] [Q&A] [answered]
Hi all, 

Currently exploring the robots plugin, however, we are currently using a UR10e. WIll the UR10 from the current robot's library work with our UR10e or will we need to add it ourselves? Is that even possible or will it work?

Thanks for your time!

**Comment by visose** [comment, marked as answer]
I understand people are using the plugin with UR5e and UR10e robots.

The generated code outputs commands that are compatible with both cb and e series (such as movel, movej, etc.), but it doesn't use commands from more modern or application specific modules.

**Reply by visose** [reply]
To answer your actual question, it looks like you might need to tweak some values in the XML file, and new meshes, as they are not exactly the same:
https://www.universal-robots.com/articles/ur/application-installation/dh-parameters-for-calculations-of-kinematics-and-dynamics/

**Reply by HmarKabob** [reply]
Thank you for the quick reply! Will check it out!
